tests/phpunit/tests/rest-api/wpRestTemplateRevisionsController.php

```diff
@@ -219,6 +251,22 @@ public function test_get_items() {
 		);
 	}
 
+
+	/**
+	 * @covers WP_REST_Template_Revisions_Controller::get_items_permissions_check
+	 * @ticket 56922
+	 */
+	public function test_should_not_return_items_with_no_permission() {
+		wp_set_current_user( 0 );
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/templates/' . self::TEST_THEME . '/' . self::TEMPLATE_NAME . '/revisions' );
+		$response = rest_get_server()->dispatch( $request );
+
+		$this->assertErrorResponse( 'rest_cannot_read', $response, 401 );
+		wp_set_current_user( self::$contributor_id );
+		$response = rest_get_server()->dispatch( $request );
+		$this->assertErrorResponse( 'rest_cannot_read', $response, 403 );
+	}
```

Comment:
Break these into two different tests.

Reply:
Fixed in 0488d6b4e5a63926cf9b9c8e3fc94eaba9ab79ac